test/fixtures/track-purchase2.json

```diff
@@ -24,13 +24,13 @@
       "currency": "USD",
       "discount": 0,
       "email": "test@iterable.com",
-      "orderId": "R550311537",
+      "order_id": "R550311537",
```

Comment:
yes -- but looking at the integration, this wouldn't be a breaking change since it looks like `orderId` was being populated directly because it was included in the properties. When people start sending that as `order_id`, it would come in as is.